Assets/General/Scripts/Story/StoryManager.cs

```diff
@@ -0,0 +1,176 @@
+/*****************************************************************************
+// Name :           StoryManager.cs
+// Author :         Charlie Polonus
+// Created :        11/6/2024
+// Description :    Handles the story beats and story beat events for during
+                    the game
+*****************************************************************************/
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class StoryManager : MonoBehaviour
+{
+    // The singleton StoryManager
+    public static StoryManager main;
```

Comment:
Although not a rule, most other instances we named Instance not main. Just trying to maintain consistency